### PR TITLE
Added many plugins

### PR DESCRIPTION
**Pull request by Loup-Garou911XD**
These plugins aren't authored by me

### CODE REVIEW

**Comment by Loup-Garou911XD** [comment]
Take your time for review

File: plugins/minigames.json [hunk 1]
```diff
@@ -97,25 +230,6 @@
           "md5sum": "c84b7f415de5d3e9189ee73fc0e3ce93"
         }
       }
-    },
-    "soccer": {
```
Comment (Loup-Garou911XD) [inline plugins/minigames.json:101]: Moved to line 6

**Comment by rikkolovescats** [comment]
let's make any filenames in lowercase

**Comment by Loup-Garou911XD** [comment]
Is that necessary?

**Comment by rikkolovescats** [comment]
won't break anything, but a python convention to keep filenames in lowercase, keeps them looking consistent

**Comment by Loup-Garou911XD** [comment]
Ok will do

**Comment by Loup-Garou911XD** [comment]
> won't break anything, but a python convention to keep filenames in lowercase, keeps them looking consistent
Did that,but checks are failing for some reason i don't understand

**Comment by Loup-Garou911XD** [comment]
https://github.com/Loup-Garou911XD/plugin-manager/pull/2
Have a look at this and let me know what's the error because I can't

File: plugins/utilities.json [hunk 2]
```diff
@@ -14,6 +103,7 @@
         }
       ],
       "versions": {
+        "1.2.1":null
```
Comment (rikkolovescats) [inline plugins/utilities.json:106]: @Loup-Garou911XD missing comma here

File: plugins/utilities/share_replay.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+def cprint(*args):
+    out = ""
+    for arg in args:
+        a = str(arg)
+        out += a
+    _ba.chatmessage(out)
+
+
+internal_dir = "/data/data/net.froemling.bombsquad/files/bombsquad_config/replays"
+external_dir = "/storage/emulated/0/Android/data/net.froemling.bombsquad/files/mods/replays"
```
Comment (Loup-Garou911XD) [inline plugins/utilities/share_replay.py:35]: Have to fix this

**Comment by Loup-Garou911XD** [comment]
> ci seems to be messed up for some reason, is it possible for you to run the tests locally for now?

Test for json?

**Comment by rikkolovescats** [comment]
> Test for json?

yep, that and more, check out the failed ci job

File: plugins/utilities/share_replay.py (hunk 3 above)
Comment (rikkolovescats) [reply plugins/utilities/share_replay.py:35]: `_ba.env()` might help

Comment (Loup-Garou911XD) [reply plugins/utilities/share_replay.py:35]: Ok will look at that,thanks

Comment (Loup-Garou911XD) [reply plugins/utilities/share_replay.py:35]: `ba.modutils._request_storage_permission()` can i request permission and copy files to the downloads folder?

Comment (Loup-Garou911XD) [reply plugins/utilities/share_replay.py:35]: Will the permission be revoked after the app restarts or Sumthing??

Comment (Loup-Garou911XD) [reply plugins/utilities/share_replay.py:35]: Because i don't want to expose user's internal storage to potential malware

Comment (rikkolovescats) [reply plugins/utilities/share_replay.py:35]: haven't tried it before but it sure seems interesting!

Comment (Loup-Garou911XD) [reply plugins/utilities/share_replay.py:35]: Maybe ill have an option to either copy to the directory with write perms or take the risk and copy to somewhere easily accessible

**Comment by Loup-Garou911XD** [comment]
> > Test for json?
> 
> yep, that and more, check out the failed ci job

I don't understand what's happening in it

**Comment by Loup-Garou911XD** [comment]
Bruh

**Comment by Loup-Garou911XD** [comment]
Fk this s***

**Comment by rikkolovescats** [comment]
don't worry about it, ci really isn't as intuitive currently as it should be, definitely needs work

**Comment by Loup-Garou911XD** [comment]
Ready to merge!

**Review by rikkolovescats — APPROVED** [review]

**Comment by rikkolovescats** [comment]
Thanks a lot!

**Pull request merged**